ocp_deployment/README.adoc

```diff
@@ -108,6 +109,8 @@ In case you have a public repository, use an empty string as the value for the `
 \ -p "SSH_PRIVATE_KEY=$(echo "" | base64 )"
 ----
 
+The certificate is optional, so you can just leave this parameter out, if for example you set it in the frontend, i.e. the app
```

Comment:
Could we move this below the "PRIVATE_KEY" explanation below? This should then also include a short description of what CERTIFICATE is, just as with the other described keys

Reply:
Fixed in commit: https://github.com/NUMde/compass-numapp-backend/pull/17/commits/d00e2ac36d4e63299a6c0cf5d1731f4a8468a04d